### PR TITLE
compare room

### PR DESCRIPTION
**Pull request by minhld1709**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -32,3 +32,49 @@ function onClick(){
...
+    what = a[--L];
+    while ((ax= arr.indexOf(what)) !== -1) {
+      arr.splice(ax, 1);
+    }
+  }
+  return arr;
+}
+
+var compare_house_ids = [];
+var checked_box = 0;
```
Comment (samnangcattor) [inline app/assets/javascripts/application.js]: you should  write  as this :  var compare_house_ids = [], checked_box = 0;

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -524,3 +524,72 @@ legend {
...
+}
+
+#compare_house{
+  display: none;
+  background: #12C0C0;
+  border-color: #12C0C0;
+}
+
+/* compare_house */
+
```
Comment (samnangcattor) [inline app/assets/stylesheets/custom.css.scss]: remove  this line

File: app/models/address.rb [hunk 3]
```diff
@@ -9,4 +9,6 @@ class Address < ActiveRecord::Base
   accepts_nested_attributes_for :images, allow_destroy: true
 
   enum type: [:apartment, :villa]
+
+  scope :compare_list,->address_ids{where id: address_ids.split(",")}
```
Comment (samnangcattor) [inline app/models/address.rb]: add one more space between , and ->

Comment (daothanhcam) [inline app/models/address.rb]: Missing some spaces.

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -524,3 +524,71 @@ legend {
 #fa-usd {
   margin-left: 5px;
 }
+
+.check_box_compare {
+  float: right;
+}
+
+#compare_house{
```
Comment (daothanhcam) [inline app/assets/stylesheets/custom.css.scss]: Missing space.

**Pull request closed without merging**